tests/actions/test_action_typedicts.py

```
@@ -0,0 +1,234 @@
+    )
+
+    assert session_file.exists(), f"Session file not found: {session_file}"
+
+    overloaded_typedicts = _extract_overload_typedicts_from_file(session_file, "execute")
+
+    # Get all expected TypedDicts
+    expected_typedicts = {f"{action_cls.__name__}Dict" for action_cls in BaseAction.ACTION_REGISTRY.values()}
+
+    missing = expected_typedicts - overloaded_typedicts
+    if missing:
+        assert False, "notte_browser/session.py execute() is missing overloads for:\n  - " + "\n  - ".join(
```

Ruff (0.14.8)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)